src/widget/text-print/text-print.js

```diff
@@ -22,27 +22,25 @@ class TextPrintWidget extends Widget {
     }
 
     _addWidget() {
-        const className = 'print-text-input';
-        const printElement = document.createElement( 'div' );
-        printElement.classList.add( className, 'widget' );
+        if ( !this.widget ) {
```

Comment:
The if() statement just prevents exceptions/duplicate elements in case the printify/deprintify events are not always following each other for any reason.